frontend/graceshopper/src/App.js

```diff
@@ -111,7 +110,6 @@ const App = () => {
         <Route path='/admin' element={
           <>
             <NavBar token={token}/> 
-            <Admin token={token}/>
```

Comment:
only concern is here, I think we'll need a route to /admin so that we can have a redirect there

Reply:
I added the Admin back, hopefully it allows you to push it now.